.circleci/config.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - run:
           name: server tests
           command: |
-            make test-server-coverage
+            pipenv run pytest -n 2 --cov=.
```

Comment:
Is there a reason this is set to 2 and not `auto`?

Reply:
Apparently on CircleCI it thinks there are 32 CPUs 🤦 so it runs way too many workers and a bunch of them crash and the tests fail. Currently, our CircleCI setup only provides 2 CPUs, so I hardcoded to 2

Reply:
See: https://ideas.circleci.com/ideas/CCI-I-578